### DISCUSSION TITLE
Sticky navigation !working

### DISCUSSION THREAD

**Discussion by dylanh724** [sphinx-doc/sphinx#12403] [Q&A]
Trying to get sticky nav working with sphinx_rtd_theme, but experiencing inconsistencies.

From what I can tell, if I add *any* toctree to a child doc, the sticky tree completely wipes and just sticks to the new toctree.

Is there any "additive" option so if I add toctree goodies, it just adds to it instead of replaces completely?

I couldn't find much about sticky nav, so any additional tips would help. Eg, I have a toctree nav that may look like

- Foo1
- Foo2

Within Foo1, I have no toctree, so the "sticky" nav persists

Within Foo2, I have a toctree containing:

- Bar

The output result is missing Foo1:

- Foo2
   - Bar

What I was really was expecting was Foo1 to port over:

- Foo1
- Foo2
   - Bar

**Comment by chrisjsewell** [comment]
Heya, would it not be better to ask this question over at https://github.com/readthedocs/sphinx_rtd_theme?

**Reply by dylanh724** [reply]
Ah not sure - I'll cross post there. Feel free to close this if you feel so. I figured sticky wasn't specific to a particular theme (I could easily be wrong; I'm still a newb).
